### DISCUSSION TITLE
why I can't find output after running

### DISCUSSION THREAD

**Discussion by Phantom0506** [mikel-brostrom/yolo_tracking#827] [Q&A] [answered]
I can train my video, and it can print done, but when I open the folder that I create to save output-video, it's empty! I don't know how to deal with this question and I want to know how to work it out, thank you!

**Comment by mikel-brostrom** [comment, marked as answer]
`python track.py --save-vid`

**Reply by Phantom0506** [reply]
Thanks! I solved this problem! And I want to know how to keep the result-video having sound. I check the result-video but find it's mute, I don't know how to fix it

**Reply by mikel-brostrom** [reply]
We don't support audio. But maybe the audio can be copied from one file to another. Please, mark this as answered :smile:

**Reply by Phantom0506** [reply]
Thank you very much！
